net/mosquitto/Portfile

```diff
@@ -78,6 +80,7 @@ pre-test {
     reinplace "s|\\./09-util-utf8|#./09-util-utf8|" \
         ${workpath}/build/test/lib/Makefile
     }
+    puts '*** NB: Tests require that default Python version is 2.x.'
```

Comment:
The following works contingent on ${prefix}/bin/python2 existing:
```tcl
    if {[file executable ${prefix}/bin/python2]} {
        fs-traverse dir ${workpath}/build/test {
            if {[string match "*.py" ${dir}]} {
                reinplace -E "s|^#!/usr/bin/env python$|#!${prefix}/bin/python2|" $dir
            }
        }
    }
```

Reply:
Also, use `${build.dir}` instead of `${workpath}/build`. The fact that it is `${workpath}/build` is an implementation detail of the cmake portgroup that you shouldn't rely on.

Also, don't use `puts` in a Portfile. If you need to print a message that can't wait until `notes` are printed after the port is installed, use `ui_msg`, but doing so should be minimized.

Reply:
@ryandesign mosquitto Portfile has been updated per your comments.  Thank you!